src/main/resources/application.properties

```diff
@@ -284,4 +284,5 @@ kafka.topics.user.bulk.upload.group=userBulkUpload
 org.channel.delimitter=_
 org.search.response.default.limit=100
 lms.system.settings.verified.profile.fields.path=v1/system/settings/get/verifiedProfileFields
-user.bulk.upload.group.value=GROUP A,GROUP B,GROUP C,GROUP D,Contractual Staff,Others
\ No newline at end of file
+user.bulk.upload.group.value=GROUP A,GROUP B,GROUP C,GROUP D,Contractual Staff,Others
+assessment.primary.category.params=Course,Program,Standalone Assessment
```

Comment:
rename this as - content.search.primary.category.filter=Course,Program,Standalone Assessment,Blended Program

update the variable name accordingly...